pkg/resource/plugin/plugin.go

```diff
@@ -229,7 +229,7 @@ func execPlugin(bin string, pluginArgs []string, pwd string) (*plugin, error) {
 	// Flow the logging information if set.
 	if logging.LogFlow {
 		if logging.LogToStderr {
-			args = append(args, "--logtostderr")
+			args = append(args, "-logtostderr")
```

Comment:
All the plugins were using `flag` directly (and hence calling `flag.Parse()`), and this is the preferred form for options there.

Reply:
:+1: